goserver/server/server.go

```diff
@@ -37,9 +48,18 @@ func (s *Server) handleLaunch(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "missing iss in URL query parameters", http.StatusUnauthorized)
 		return
 	}
-	if iss[0] != s.AuthorizedFHIRURL {
+	if iss[0] != s.authorizedFHIRURL {
 		http.Error(w, fmt.Sprintf("unauthorized iss %s", iss[0]), http.StatusUnauthorized)
 		return
 	}
+
+	sess, err := s.sm.New(w, r)
+	if err != nil {
+		http.Error(w, "cannot create session", http.StatusInternalServerError)
```

Comment:
We're losing the actual error details here. I agree we shouldn't return them to the user but we should log them to server logs so we can investigate. We also should probably have a single generic error message for any internal errors to make it clear that it's an issue on our end and not something the user did wrong.

Can you file an issue and add a TODO(Issue#) to figure out consistent error handling (e.g. maybe have a library that writes all top level errors to logs/outputs Cloud Logging metrics and transforms internal errors properly).

Reply:
TODO added.